MovieApp/MovieApp/App/Features/MovieSearch/MoviesViewController/MoviesViewController.swift

```diff
@@ -1,27 +1,33 @@
 import UIKit
 
 class MoviesViewController: UIViewController {
-    
+
     let rowHeight: CGFloat = 142
-    
+
     var logoImageView: UIImageView!
     var tableView: UITableView!
     var movies = [Movie]()
-    
-    private var presenter: MainViewPresenter!
-    
-    convenience init(presenter: MainViewPresenter) {
-        self.init()
 
+    private var presenter: MoviesViewPresenter!
+
+    convenience init(presenter: MoviesViewPresenter) {
+        self.init()
+        
         self.presenter = presenter
     }
-    
+
     override func viewDidLoad() {
         super.viewDidLoad()
         
-        movies = presenter.movies
+        presenter.setMoviesViewDelegate(moviesViewDelegate: self)
+        presenter.fetchMovies()
+        
```

Comment:
delete extra line

Reply:
shouldn't we have empty line after super.viewDidLoad()?

Reply:
yep

Reply:
I see, you suggested to remove line after presenter.fetchMovies().

I added that line because of separation of logical groups of code but will remove it.